tests/blake2b.cc

```diff
@@ -38,31 +38,19 @@ using namespace std;
 
 // -----------------------------------------------------------------------------
 
-// API EXAMPLE HEX START
-void
-print_hex_ln(size_t bytes_len, uint8_t* bytes)
-{
-  for (int i = 0; i < bytes_len; ++i) {
-    printf("%02x", bytes[i]);
-  }
-
-  printf("\n");
-}
-// API EXAMPLE HEX END
-
 TEST(ApiTestSuite, ApiTest)
 {
-  // API EXAMPLE CODE START
+  // ANCHOR(example)
   // Reserve memory for a 64 byte digest, i.e.,
-  // for a BLAKE2B run with full 512-bit output.
+  // for a BLAKE2b run with full 512-bit output.
   uint32_t output_len = 64;
```

Comment:
So you just want to keep it like this? It's fine with me. But maybe note the changes we'd like to get done somewhere.

Reply:
Ah, yeah. I wanted to make a pass over all (new and old) examples later. Will open an issue.

Reply:
https://github.com/cryspen/hacl-packages/issues/285